Watcher/v6/_summit.yaml

```diff
@@ -233,8 +233,11 @@ rules:
     poll_interval: 1.0
 - classname: MTForceError
   configs:
-  - force_error_axial: 10.0
-    force_error_tangent: 25.0
+  - force_error_axial: 15.0
+    force_error_tangent: 50.0
+    max_num_axial: 10
+    max_num_tangent: 2
+    threshold_mtmount_acceleration: 0.05
```

Comment:
These all match the default values so there really is no need to include them. Either remove them here or remove the default values from the ts_watcher rule config schema.

Reply:
I loosen the condition to reflect the decayed hardware performance based on the recent break-out test. This will have different values from the defaults. Updated in the commit: [b90d290](https://github.com/lsst-ts/ts_config_ocs/pull/352/commits/b90d29042961bae437282667b7ffe53137980f4c). Thanks!